var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -124,6 +124,7 @@ def configure_args(self):
             args.append("--disable-html")
 
         if "+squashfuse" in self.spec:
-            args.append("--with-libsquashfuse=yes")
+            squash_bin="{0}".format(self.spec["squashfuse"].prefix.bin)
```

Comment:
Hmm, I think it may need to be as follows? Note bin dropped.

```python3
    squashfuse_dir="{0}".format(self.spec["squashfuse"].prefix)
    args.append("--with-libsquashfuse={0}".format(squashfuse_dir))
```

Reply:
j-ogas
You are right. Unfortunately build did not fail on error and used the system library instead. Changed as per your suggestion